rust-bot/src/main.rs

```diff
@@ -171,7 +207,7 @@ fn mute_users_without_role_permset(
         },
     ]
 }
-
+#[instrument(skip(ctx))]
```

Comment:
I see some skips here. Does that mean all of `Message` is included in the event? Particularly the message content.

Reply:
The `instrument` call creates a span and passes all of the args to the span using the `fmt::Debug` implementation for each arg. 

Serenity's context type doesn't have debug implemented, so for now I'm just skipping that specific arg. The other args still get included

Reply:
ok, I think we should ignore message for now then and try to figure out how to safelist message fields into the event instead so that we aren't recording message content into a third party system